recipe-server/normandy/recipes/models.py

```diff
@@ -560,7 +560,7 @@ def get_absolute_url(self):
         return reverse('action-detail', args=[self.name])
 
     def compute_implementation_hash(self):
-        return hashlib.sha1(self.implementation.encode()).hexdigest()
+        return sri_hash(self.implementation.encode(), url_safe=True)
```

Comment:
Ah, got it. Can you leave a comment explaining that an SRI-style hash is intentional, then?